ddtrace/contrib/algoliasearch/patch.py

```diff
@@ -37,7 +42,7 @@ def get_version():
 
 
 def patch():
-    if algoliasearch_version == (0, 0):
+    if algoliasearch_version == V0:
```

Comment:
Is this really `V0` or just `<V1`

Reply:
I think `(0, 0)` was being used here as a sentinel. I'll take another look - maybe it's possible to remove it entirely.